csp-guru/datapackage.json

```diff
@@ -19,5 +169,11 @@
             "email": "merce.larbordena@usys.ethz.ch",
             "role": "author"
         }
-    ]
+    ],
+    "licenses":[
+      {
+  "name": "CC BY-NC 4.0",
```

Comment:
Something is completely wrong with the indentation here. Make sure you use 4 spaces per indentation level as this was used in the rest of the file and then bring it on the correct indentation level. An editor like Atom can do that automatically for you if you don't want to do it manually.